packages/slice-machine/src/features/builder/StaticFieldsInfoDialog.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  Box,
+  Dialog,
+  DialogActionButton,
+  DialogActions,
+  DialogContent,
+  DialogHeader,
+  ScrollArea,
+  Text,
+  Video,
+} from "@prismicio/editor-ui";
+import { ReactNode, useState } from "react";
+
+interface StaticFieldsInfoDialogProps {
+  onClose: () => void;
+  trigger?: ReactNode;
+}
+
+export function StaticFieldsInfoDialog(props: StaticFieldsInfoDialogProps) {
+  const { onClose, trigger } = props;
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleOpenChange = (open: boolean) => {
+    if (!open) {
+      dismissDialog();
+    } else {
+      setIsOpen(open);
+    }
+  };
+
+  function dismissDialog() {
+    setIsOpen(false);
+    onClose();
+  }
+
+  return (
+    <Dialog
+      open={isOpen}
+      onOpenChange={handleOpenChange}
+      trigger={trigger}
+      size="small"
+    >
+      <DialogHeader title="Static zone" />
+      <DialogContent>
+        <ScrollArea>
+          <Box as="article" flexDirection="column" padding={16} gap={4}>
+            <Text sx={{ marginBottom: 4 }} color="purple9" variant="smallBold">
+              About
+            </Text>
+            <Text variant="h3">What is the Static zone</Text>
+            <Text color="grey11">
+              The static zone in Prismic page types contain fields that are
+              always present on the page. Use it for essential information like
+              a page's title, page design settings or metadata. Unlike slices,
+              which can be added or removed, static zone fields remain fixed.
+            </Text>
+            <Video
+              src="https://res.cloudinary.com/dmtf1daqp/video/upload/v1723540305/DEV_TOOLS/STATES/Explication_vide%CC%81o_zone_statique_phibeq.mp4"
+              sizing="contain"
+              autoPlay
+              loop
+            />
+          </Box>
+        </ScrollArea>
+      </DialogContent>
+      <DialogActions>
+        <DialogActionButton
+          size="medium"
+          sx={{ flexGrow: 1 }}
+          onClick={dismissDialog}
+          data-testid="static-fields-info-dialog-cta"
```

Comment:
Yeah it's not being used, but IMO test ids for E2E are better, that way we don't keep breaking the tests if we want to update a simple label, only the full flow matters.

Reply:
I don't agree to that, I prefer to have text when text is part of your product. It's our case on our side, with exact, it allows us to prevent someone to make a change of text without wanting to. I remember one time when I added a "s" because I wanted to do CTRL + s to save and e2e tests broke to warn me about it. 
When your text is part of your product, I would always favor text.